test/project.py

```diff
@@ -389,3 +388,9 @@ def patch_cartridge_version(project, new_version):
 
     with open(os.path.join(project.path, 'init.lua'), 'a') as f:
         f.write(new_version_code)
+
+
+def rewrite_project_file(project, project_filepath, filepath):
```

Comment:
It's not clear that does it mean - seems arguments should have more verbose names.

Why do you rewrite content of file... Why can't you drop previous file and copy new one?

Reply:
Refactored.